src/subscript/sunsch/time_vector.py

```diff
@@ -0,0 +1,409 @@
+import datetime
```

Comment:
Add a comment at the top that this file is due to be removed by November 2021, when OPM 2021.10 is out.